root/account/Preferences.js

```diff
@@ -21,11 +23,12 @@ type Props = {|
   ...PreferencesFormPropsT,
 |};
 
-const Preferences = withCatalystContext(({$c, ...props}) => (
+const Preferences = withCatalystContext(({$c, ...props}: Props) => (
   <UserAccountLayout
+    // $FlowFixMe We won't ever reach this template if $c.user does not exist
```

Comment:
You can tell Flow it always exists by changing the prop to `+$c: {user: EditorT} & CatalystContextT,`.